### PR TITLE
Styling for industry averages

### PR DESCRIPTION
**Pull request by kacyhulme**
Fixes [this To Do](https://3.basecamp.com/3097764/buckets/19037960/todos/3099725492)

### Testing notes

1. Visit the ad spend calc
2. Scroll to bottom
3. Should match [designs](https://projects.invisionapp.com/d/main/default/#/console/20628379/433566411/inspect) 

### Other information
This is working, however, I should probably refactor it.

### CODE REVIEW

**Comment by kacyhulme** [comment]
@codegoalie I got this working. It could use some refactoring. Open to your feedback.

File: src/Averages.js [hunk 1]
```diff
@@ -0,0 +1,78 @@
+import React from "react";
+import styled from "styled-components";
+
+const Averages = ({ type }) => {
+  const highAvgs = [
```
Comment (codegoalie) [inline src/Averages.js:6]: One thing to keep in mind is the single responsibility principle. In short, each aspect of the software should be responsible for one thing. A code smell to indicate that you might be working against that principle is a boolean parameter that only exists to toggle between 2 outcomes of the component. The `type` prop (although it's a string) is like that here. This component is responsible for assembling the `div`s of both the high and low averages: 2 responsibilities.

One potential solution would be do define 2 components `HighAverages` and `LowAverages`. These each would be very similar to this component without the switching logic.

A more DRY solution could also be to move the row content constants out of this component and into the `IndustryAverages` component. Then this component does only one thing: takes any JS object and returns a list of `div`s with the `key` and `value`of each attribute in the object.

There are probably other solutions too. Both of the two I've listed have their pros and cons.

**Review by codegoalie — CHANGES_REQUESTED** [review]
This looks good.

My one comment is about the overall organization of the code. Probably not something that needs to be addressed now, but something to consider in the future.

The only issue I saw is that the alternating row background is not working.

image

File: src/Averages.js (hunk 1 above)
Comment (kacyhulme) [reply src/Averages.js:6]: @codegoalie I tried the first approach, breaking out the Low and High processes into two different components. [New branch. ](https://github.com/growthtools/ad-spend-calc/pull/20)

**Review by codegoalie — COMMENTED** [review]
@kacyhulme I fixed some formatting and linting issues and then tweaked the subheading language a bit.

Also, the margins and padding didn't seem to fit the design. Check out the css changes I made and if those look good to you, I'm good to merge.

**Pull request merged**